test/snapshot/__snapshots__/attributes.test.js.snap

```diff
@@ -335,7 +324,18 @@ Program {
           ],
           "final": false,
           "kind": "classconstant",
-          "nullable": false,
+          "nullable": [
+            AttrGroup {
+              "attrs": [
+                Attribute {
+                  "args": [],
+                  "kind": "attribute",
+                  "name": "B",
+                },
+              ],
+              "kind": "attrgroup",
+            },
```

Comment:
This does not make sense either, nullable should be a bool value? looks like attrGroups and nullable are switched around somewhere